extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java

```diff
@@ -160,7 +167,8 @@ public void dispose() {
         if (lightId != null) {
             HueBridgeHandler bridgeHandler = getHueBridgeHandler();
             if (bridgeHandler != null) {
-                getHueBridgeHandler().unregisterLightStatusListener(this);
+                this.bridgeHandler.unregisterLightStatusListener(this);
+                this.bridgeHandler = null;
```

Comment:
You are right, I have not realized the line above the if clause and missed that local member.